plugins/outputs/newrelic/README.md

```diff
@@ -10,10 +10,14 @@ Telegraf minimum version: Telegraf 1.15.0
 
 ```toml
 [[outputs.newrelic]]
-  ## New Relic Insights API key
-  insights_key = "insights api key"
+  ## The 'insights_key' parameter requires a NR license key.
+  ## New Relic recommends you create one
+  ## with a convenient name such as TELEGRAF_INSERT_KEY.
+  ## reference: https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#ingest-license-key
+  insights_key = "New Relic License Key Here"
 
   ## Prefix to add to add to metric name for easy identification.
+  ## This is very useful if your metric names are ambiguous.
```

Comment:
Please add this in the `SampleConfig` as well.

Reply:
I have the change made in the SampleConfig method newrelic.go.  Which file are you referring to?

Reply:
newrelic.go, this specific line doesn't exist in there.